P5/Source/Specs/citeData.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="header" ident="citeData">
+  <gloss versionDate="2020-02-03" xml:lang="en">citation data</gloss>
+  <desc versionDate="2020-02-03" xml:lang="en">specifies how information may be extracted from citation structures.</desc>
+  <classes>
+    <memberOf key="att.global"/>
+    <memberOf key="att.citeStructurePart"/>
+  </classes>
+  <content>
+    <empty/>        
+  </content>
+  <attList>
+    <attDef ident="property" usage="req">
+      <gloss versionDate="2020-02-03" xml:lang="en">property</gloss>
+      <desc versionDate="2020-02-03" xml:lang="en">A URI indicating a property definition.</desc>
+      <datatype>
+        <dataRef name="anyURI"/>
```

Comment:
While I'd always advise for using URIs, I wonder if we should enforce it here. In your own example in #1957 you put in "dc:title" as property – and I think that's ok since this mechanism can potentially be used to _label_ all sorts of features of a document.

Reply:
Well, "dc:title" is (most likely) a URI. If I used it, I'd have a `<prefixDef>` explaining how to expand it.